OSDOCS-18372: DITA abstracts — KMM (#117165)
openshift/openshift-docs: modules/kmm-setting-soft-dependencies-between-kernel-modules.adoc

[id="kmm-setting-soft-dependencies-between-kernel-modules_{context}"]
= Set soft dependencies between kernel modules

[role="_abstract"]
Soft dependencies require kernel modules to load in a specific order even when they do not share symbols. You can declare these dependencies in the `Module` CR with the `modulesLoadingOrder` field.

> `depmod` is usually not aware of soft dependencies, and they do not appear in the files it produces. For example, if `mod_a` has a soft dependency on `mod_b`, `modprobe mod_a` will not load `mod_b`.

Reviewer: Maybe consider revising line 12. "`depmod is usually not aware` comes across as anthropomorphic. 

If depmod is a tool...

```suggestion
The `depmod` tool does not recognize soft dependencies, and soft dependencies do not appear in the files it produces. For example, if `mod_a` has a soft dependency on `mod_b`, `modprobe mod_a` will not load `mod_b`.
```
